cmd/libs/go2idl/types/types.go

```diff
@@ -56,6 +56,9 @@ const (
 	// We then need "Alias" as a way for us to say that Bar *is* a Foo.
 	Alias Kind = "Alias"
 
+	// DeclarationOf is a declaration of a type.
```

Comment:
Sorry, one last request-- can we change the comment as below, and also move this down in the list, between "Func" and "Unknown"?

// DeclarationOf is different from other Kinds; it indicates that instead of representing an actual Type, the type is a declaration of an instance of a type. E.g., a top-level function, variable, or constant. See the comment for Type.Name for more detail.